apps/web/app/modules/Dashboard/Courses/CourseCard.tsx

```diff
@@ -47,8 +60,19 @@ const CourseCard = ({
             <div dangerouslySetInnerHTML={{ __html: description }} />
           </span>
         </div>
-        <Button className="w-full mt-auto">
-          {isAdmin || enrolled ? "View" : "Enroll"}
+        <Button
+          variant={enrolled ? "secondary" : "primary"}
+          className="mt-auto w-full"
+        >
+          {enrolled ? (
+            <span className="flex gap-x-2 items-center">
+              <Icon name="ArrowRight" className="w-4 h-4 text-white" /> Continue
+            </span>
+          ) : isAdmin ? (
+            "View"
+          ) : (
+            "Enroll"
+          )}
```

Comment:
What do you think about extracting this into a separate component, or at least a function, to get rid of this nested ternary?